container/nginx/nginx.conf

```diff
@@ -17,7 +17,7 @@ http {
 
     log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
                       '$status $body_bytes_sent "$http_referer" '
-                      '"$http_user_agent" "$http_x_forwarded_for"';
+                      '"$http_user_agent" "$http_x_forwarded_for" "$http3"';
```

Comment:
Keep in mind this log format is not used.

Reply:
I did miss that 😅 . thanks!